website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,407 @@
+
+It is better than the other modules because its input data is readily available and there is considerable overlap with hybrid growth modules and WOFOST modules. For a detailed explanation and better understanding you can refer to the [RothC guide](https://www.verdeterreprod.fr/wp-content/uploads/2020/05/RothC_guide_WIN.pdf).
+
+### Inputs:
+
+-   Time Step (Monthly).
+-   Pool 1: Input of plant residue.
+-   Pool 2: Input of farmyard manure.
+-   Pool 3: Total organic carbon.
+-   Pool 4: Microbial biomass carbon.
+-   Rainfall.
+-   Open pan evaporation.
+-   Mean air temperature.
+-   Clay content of soil.
+-   Decomposability of incoming plant material.
+-   Soil Cover.
+-   Depth of soil layer sampled.
+
+### Output:
+
+-   Total Organic Carbon.
+-   Microbial Biomass Carbon.
+
+|![Pictorial representation of RothC Module.](../static/img/RothC-module-case-study.png) |
+|:--:|
+| Pictorial representation of RothC Module. |
+
+## Chapman Richards Module
+
+The Chapman Richards module is a tier-3 module. It is used for the estimation of greenhouse gas emissions and removals due to changes in biomass carbon for forests using Chapman Richards equation. We can find aboveground and belowground biomass using this module.
+
+```
+AGC = * [ 1 -  e ^ { -k *Age } ] ^ { ( 1 / (1-m) }
+```
+
+-   **AGC**: Above-ground stand carbon (Tonnes carbon per hectare).
+-   **Max**: Asymptote – maximum peak carbon yield (Tonnes carbon per hectare).
+-   **k**: Parameter used in modelling tree growth (dimensionless).
+-   **Age**: Age of the forest (years).
+-   **m**: Parameter used in modelling tree growth (dimensionless).
+
+This module provides us the estimation of Aboveground biomass on the basis of the age of forests. Belowground biomass is calculated as a ratio from aboveground biomass. This equation is also used within the Forest Growth module. There is a separate repository for the [Chapman Richards module](https://github.com/moja-global/FLINT.Module.Chapman_Richards). It works on a monthly time step.
+
+### Input
+
+-   Time Step (Monthly)
+-   Dead Organic Matter
+-   Soil organic matter
+-   Pool 1 : Non-CO2 gases (The selection of carbon pools or non-CO2 gases for estimation will depend on the significance of the pool and tier selected for each land-use category)
+-   Forest Age
+-   Forest Type
+-   Forest Cover
+-   Soil Carbon
+-   Spatial Location of Forest
+
+### Output
+
+-   Above-ground biomass
+-   Below-ground biomass
+
+For Forest Land complete documentation refer to the [Forest land documentation](https://www.ipcc-nggip.iges.or.jp/public/2006gl/pdf/4_Volume4/V4_04_Ch4_Forest_Land.pdf) which is a part of Volume 4: Agriculture, Forestry and Other Land Use.
```

Comment:
I haven't read this document, but from a first impression it doesn't seem specific to Chapman-Richards.